ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -308,6 +303,18 @@ void OwInterface::antennaFaultCallback
   updateFaultStatus (msg->value, m_panTiltErrors, "ANTENNA", "AntennaFault");
 }
 
+static double normalize_degrees (double angle)
+{
+  static double pi_degrees = R2D * M_PI;
+  while (angle > pi_degrees) {
+    angle -= 2.0 * pi_degrees;
+  }
+  while (angle < -pi_degrees) {
+    angle += 2.0 * pi_degrees;
+  }
+  return angle;
+}
```

Comment:
[optional] This could be done in the same form as in utils.py with `fmod`.

Reply:
Agreed, done.